metaflow/flowspec.py

```diff
@@ -28,7 +28,9 @@
 
 from .datastore.inputs import Inputs
 
-INTERNAL_ARTIFACTS_SET = set(["_foreach_values", "_unbounded_foreach"])
+INTERNAL_ARTIFACTS_SET = set(
```

Comment:
Should we also exclude `_control_task_is_mapper_zero`?

Reply:
Done